app/api/discount_codes.py

```diff
@@ -97,7 +97,7 @@ def before_create_object(self, data, view_kwargs):
     def after_create_object(self, discount, data, view_kwargs):
         if data['used_for'] == 'event' and 'events' in data:
             for event_id in data['events']:
-                event = safe_query(self, Event, 'id', event_id, 'event_id')
+                event = safe_query(Event, 'id', event_id, 'event_id')
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [No value for argument 'parameter_name' in function call](https://app.codacy.com/manual/fossasia/open-event-orga-server/pullRequest?prid=5526478)